dags/hivemind_etl_helpers/tests/unit/test_telegram_daily_summarize.py

```
@@ -0,0 +1,155 @@
+                        reactors=[],
+                        repliers=[],
+                    ),
+                ],
+            }
+        )
+
+        for key, summary in summaries.items():
+            self.assertEqual(key, date(2024, 6, 6))
+            self.assertIsInstance(summary, str)
+
+    def test_summarize_single_day_single_document(self):
```

Ruff

Redefinition of unused `test_summarize_single_day_single_document` from line 22

(F811)